packages/rum/README.md

```diff
@@ -376,10 +376,10 @@ init(configuration: {
 
 ### Browser and Browser Premium sampling configuration
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Move this whole section into the new sampling guide. This way, everything is documented at the same place. We can just link to it directly.

Reply:
i like that idea